File: DotNet/Apps/ConfMgrGUI/OpsTrackingModel/src/OpsTrackingModel/InterOpTester.java
package OpsTrackingModel;

import java.io.Serializable;
import java.util.Date;

//import com.gigaspaces.annotation.pojo.SpaceId;
//import com.gigaspaces.annotation.pojo.SpaceProperty;
//import com.gigaspaces.annotation.pojo.SpaceRouting;

public class InterOpTester implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private long _id = -1;
    private Date  _myDate = null;
    
    
  //  @SpaceId    //	Sets this field (ID) to be used in the construction
 //   @SpaceRouting
 //   @SpaceProperty(nullValue="-1")
	public long get_id() {
		return _id;
	}
	public void set_id(long _id) {
		this._id = _id;
	}
	public Date get_myDate() {
		return _myDate;
	}
	public void set_myDate(Date date) {
		_myDate = date;
	}
}

File: DotNet/Apps/ConfMgrGUI/OpsTrackingModel/InterOpTester.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OpsTrackingModel
{
    [System.Serializable]
    public class InterOpTester
    {
        private const System.Int32 _nullNumberValue = -1;

        
        private System.Int32 _id;
        private System.DateTime? _myDate;

        public System.DateTime? MyDate
        {
            get { return _myDate; }
            set { _myDate = value; }
        }

        public System.Int32 Id
        {
            get { return _id; }
            set { _id = value; }
        }

        public InterOpTester()
        {
            this._id = _nullNumberValue;
            this._myDate = null;
        }
    }
}
